packaging/linux/postinstall

```diff
@@ -11,6 +11,8 @@ unit_file=/usr/lib/systemd/system/device-agent-helper.service
 mkdir -p "${config_dir}"
 
 sed -i "s%@@CONFIG_DIR@@%${config_dir}%" "${unit_file}"
+systemctl daemon-reload
+systemctl restart device-agent-helper
```

Comment:
This service must be installed as root to leverage wireguard, so I'm quite certain that we cannot automate this.

I'd suggest instead of automating we give explicit instructions that people need to reload systemd file config for the naisdevice service. 

I say this due to the `mkdir -p ${config_dir}" at the top of the file...

Reply:
Looking closer, the `product_serial` copy is made lower down, so maybe instead add a `chown "$USER.$USER" "${config_dir}" then.

In which case the `systemctl` commands will work just fine.

Reply:
fpm already sets a flag to reload and restart the systemd service when we include a systemd file, so no need to do that here.

Reply:
As long as the device-agent-helper has a folder to watch it should be happy. This script is run by root.

Reply:
> Looking closer, the `product_serial` copy is made lower down, so maybe instead add a `chown "$USER.$USER" "${config_dir}"` then.
> 
> In which case the `systemctl` commands will work just fine.

This is already in the script

Reply:
> As long as the device-agent-helper has a folder to watch it should be happy. This script is run by root.
@Sechmann I don't see how that addresses the concern that on first-install the `"${config_dir}"` will be owned by `root`....

Reply:
@x10an14 : The last line of the script does a `chown` for the entire config dir.

Reply:
`${config_dir}` will be owned by the logged in user after first install. See last line in `postinstall`

Reply:
Ah, I missed that, sorry. Thanks!